showdown/battle_modifier.py

```diff
@@ -336,14 +336,21 @@ def cureteam(battle, split_msg):
 
 
 def weather(battle, split_msg):
+    weather_name = normalize_name(split_msg[2].split(':')[-1].strip())
+    logger.debug("Weather {} started".format(weather_name))
+
+    if weather_name == "none":
+        battle.weather = PersistentCondition()
+        return
+
```

Comment:
Need some tests to show the expected behaviour here - the PersistentCondition object isn't even imported in this module.